### PR TITLE
Submodule integration

### PR DESCRIPTION
**Pull request by Lakshay983**
We now use the KiCAD library repository as our symbol library.

### CODE REVIEW

File: .gitignore
Comment (LeftistTachyon) [inline .gitignore:1]: Thanks for the correct `.gitignore` :tada:

File: Chassis/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-Chassis-backups
+# Backups
+Chassis-backups/*
```
Comment (LeftistTachyon) [inline Chassis/.gitignore:2]: How about the lock file?

File: Chassis/Chassis.kicad_sch
Comment (LeftistTachyon) [inline Chassis/Chassis.kicad_sch:1]: Did you change the schematic at all? I don't see anything noteworthy.

**Review by LeftistTachyon — COMMENTED** [review]
Nice work, getting the submodules to work! Just lmk what new members need to do to set it up/edit the wiki & `README.md` yourself to reflect the changes :D

**Review by LeftistTachyon — CHANGES_REQUESTED** [review]
Forgot to check this box whoops xdddér

File: Chassis/.gitignore (hunk 1 above)
Comment (Lakshay983) [reply Chassis/.gitignore:2]: I put that into the KiCAD-Library gitignore

File: Chassis/Chassis.kicad_sch
Comment (Lakshay983) [reply Chassis/Chassis.kicad_sch:1]: I just added the symbols again. The old symbols were from the old library we deprecated. It's the exact same except the links won't be broken

File: Chassis/.gitignore (hunk 1 above)
Comment (LeftistTachyon) [reply Chassis/.gitignore:2]: You right, I blind

File: Chassis/Chassis.kicad_sch
Comment (LeftistTachyon) [reply Chassis/Chassis.kicad_sch:1]: Gotcha, that makes sense.

**Review by LeftistTachyon — APPROVED** [review]
`lgtm`

**Pull request merged**